src/common/AudioStreamBuilder.cpp

```diff
@@ -94,14 +94,19 @@ Result AudioStreamBuilder::openStream(AudioStream **streamPP) {
 }
 
 Result AudioStreamBuilder::openStreamInternal(AudioStream **streamPP) {
+    static bool versionLogged = false;
```

Comment:
Do we need this? I'm not too big of a fan of static variables like this and openStream() shouldn't realistically be called that often. Also, this is a LOGI and not something like LOGE

Reply:
When I run the unit tests it tries to trigger a deadlock by opening and closing rapidly many times. These version logs spew into the test output and can be very spammy. Maybe I could add a compiler flag that disables this log. Then we could just disable it for the tests. That seems cleaner.  Something like:

    #ifndef OBOE_DISABLE_VERSION_LOG
        LOGI(blah blah);
    #endif

Reply:
Done